lib/src/textArea/textArea.component.html

```diff
@@ -0,0 +1,30 @@
+<div class="form-group text-area" [ngClass]="className">
+    <div class="input-field" [class.has-error]="error">
+        <ng-container *ngIf="label">
```

Comment:
We can and it will work. But i chose to not have it for consistency because the use the *ngIf on ng-container in the previous components here.

Reply:
I agree with @mario-subo .. using `ngIf` on the label is a clear solution. You only need the container when you have multiple items bound to the same `ngIf` logic, therefore, it will eliminate the repetition